packages/utils/node-resolver-core/src/NodeResolver.js

```diff
@@ -32,6 +35,26 @@ type Module = {|
   filePath?: FilePath,
 |};
 
+const parseFilePath = (
```

Comment:
Not entirely sure if this is the correct place for this.

I do believe we should pass queryparams to the resolver, in case it should resolve to some remote asset like `deno` does for example... But maybe we should make the resolverrunner handle this and pass it as an argument to each resolver so they can opt in to caring about it or not?